doc/generated/libpmemfile-posix.3

```diff
@@ -36,13 +36,10 @@
 \f[B]libpmemfile\-posix\f[] \- user space persistent memory aware file
 system API
 .SH SYNOPSIS
-.IP
-.nf
-\f[C]
-#include\ <libpmemfile\-posix.h>
-cc\ \-std=gnu99\ ..\ \-lpmemfile\-posix\ \-lpmemobj
-\f[]
-.fi
+.PP
+\f[C]#include\ <libpmemfile\-posix.h>\f[]
+.PP
+\f[C]cc\ \-std=gnu99\ ..\ `pkg\-config\ libpmemfile\-posix\ \-\-cflags\ \-\-libs`\f[]
```

Comment:
What C99 features? Cuz for C++ users, mostly they would imply C++11, e.g. stdint header appeared in C++ first in C++11, which isn't that old actually.